packages/block-library/src/preformatted/styles.native.scss

```diff
@@ -0,0 +1,27 @@
+.wpBlockPreformatted {
+	padding-left: 16;
+	padding-right: 16;
+	padding-top: 12;
+	padding-bottom: 12;
+	background-color: $gray-light;
+	border-top-left-radius: 4;
+	border-top-right-radius: 4;
+	border-bottom-left-radius: 4;
+	border-bottom-right-radius: 4;
+}
+
+.wpRichText {
+	background-color: $gray-light;
+}
+
+.wpBlockPreformattedDark {
+	padding-left: 16;
+	padding-right: 16;
+	padding-top: 12;
+	padding-bottom: 12;
```

Comment:
Rather than repeating these in both color schemes, which leads to things like this one missing the border-radius on dark mode, we can leverage SASS for more than just color variables. This seems to work fine:

```scss
%wpBlockPreformattedShared {
	padding: 12px 16px;
	border-radius: 4px;
}

.wpBlockPreformatted {
	@extend %wpBlockPreformattedShared;
	background-color: $gray-light;
}

.wpRichText {
	background-color: $gray-light;
}

.wpBlockPreformattedDark {
	@extend %wpBlockPreformattedShared;
	background-color: $gray-100;
}

.wpRichTextDark {
	background-color: $gray-100;
}
```

Reply:
Also, I think it'd be a bit clearer to be explicit about light mode as well: `%wpBlockPreformatted`, `.wpBlockPreformattedLight`, `.wpBlockPreformattedDark`,...

Reply:
In this case, the `Dark` variant doesn't need to re-define properties other than the colors we want to override. `getStylesFromColorScheme` will merge all other properties.

Reply:
Oh that's right. I got confused because I see `.wpBlockPreformattedDark` redefining the same paddings, but not the border radius.

Reply:
> %wpBlockPreformattedShared

Again I don't think this work with our transformer in RN.

Reply:
> getStylesFromColorScheme will merge all other properties.

Nice!